server/s3_data_usage.cc

```diff
@@ -125,8 +125,13 @@ std::string S3DataUsageCache::generate_cache_key(
 
 std::string get_server_id() {
   s3_log(S3_LOG_DEBUG, "", "%s Entry\n", __func__);
-  s3_log(S3_LOG_DEBUG, "", "%s Exit\n", __func__);
-  return "1";
+  extern struct m0_config motr_conf;
+  // static std::string s3server_ID = motr_conf.mc_process_fid;
+  // FID is not properly integrated in dev env, so will be using Motr EP
+  // address:
+  static std::string s3server_ID = motr_conf.mc_local_addr;
+  s3_log(S3_LOG_DEBUG, "", "%s Exit, ret %s\n", __func__, s3server_ID.c_str());
+  return s3server_ID;
```

Comment:
Minor: not sure if server ID improvement should be in this PR, unless we might want to merge only this one and close
https://github.com/Seagate/cortx-s3server/pull/1691